src/applications/vaos/new-appointment/redux/helpers/eligibility.js

```diff
@@ -204,7 +207,7 @@ export function getEligibilityChecks(eligibilityData) {
       requestPastVisitValue:
         eligibilityData.requestPastVisit?.durationInMonths || null,
       requestLimit: isUnderRequestLimit(eligibilityData),
-      requestLimitValue: eligibilityData.requestLimits.requestLimit,
+      requestLimitValue: eligibilityData.requestLimits?.requestLimit,
```

Comment:
I'm guessing this change is because the mock returns an empty array if the facility requested is a 984 facility. If we fix the mock, this might not be necessary.